rts/System/FileSystem/FileHandler.h

```diff
@@ -6,6 +6,7 @@
 #include <vector>
 #include <string>
 #include <fstream>
+#include <memory>
```

Comment:
Shouldn't this be added to some .cpp file(s) instead? Looks like no new code was added below so probably doesn't need to be in the header (gets applies everywhere the .h is included even if not all .cpp files need it)